src/main/java/in/twizmwaz/cardinal/module/modules/damageIndicator/PlayerBoundingBox.java

```diff
@@ -146,23 +145,17 @@ public void sendSpawnPackets(Player viewer) {
         if (viewers.contains(viewer.getUniqueId())) return;
         Player player = Bukkit.getPlayer(this.player);
         Location loc = player.getLocation();
-
-        List<DataWatcher.Item<?>> dataItems = new ArrayList<>();
-        dataItems.add(new DataWatcher.Item<>(DataWatcherRegistry.a.a(0), (byte) 32)); // Sets invisible
-        dataItems.add(new DataWatcher.Item<>(DataWatcherRegistry.c.a(7), 20.0F));    // Sets health
-
         int i = 0;
         for (int x = -1; x < 2; x += 2) {
             for (int z = -1; z < 2; z += 2) {
                 Packet spawnPacket = new PacketPlayOutSpawnEntityLiving(
-                        zombieID.get(i), UUID.randomUUID(), 54,                 // Entity id, UUID, and type (Zombie)
-                        loc.getX() + (x * DamageIndicator.OFFSET), loc.getY() , // X, Y
-                        loc.getZ() + (z * DamageIndicator.OFFSET),              // and Z coords
-                        0, 0, 0,                                                // X, Y and Z Motion
-                        (byte)2, (byte)0, (byte)2,                              // Yaw, Pitch and Head Pitch
-                        dataItems);                                             // Metadata
+                        zombieID.get(i++), UUID.randomUUID(), 54,             // Entity id, UUID, and type (Zombie)
```

Comment:
Just to make sure you know, the function will be passed the original value of i, and increment i after. To pass the incremented value, ++i